modules/persistent-storage-csi-olm-operator-install.adoc

```diff
@@ -24,24 +41,42 @@ To install the {FeatureName} CSI Driver Operator from the web console:
 .. Locate the {FeatureName} CSI Operator by typing *{FeatureName} CSI* in the filter box.
 
 .. Click the *{FeatureName} CSI Driver Operator* button.
+
+ifdef::restricted[]
 +
 [IMPORTANT]
 ====
 Be sure to select the *{FeatureName} CSI Driver Operator* and not the *{FeatureName} Operator*. The *{FeatureName} Operator* is a community Operator and is not supported by Red Hat.
 ====
+endif::restricted[]
 
 .. On the *{FeatureName} CSI Driver Operator* page, click *Install*.
 
 .. On the *Install Operator* page, ensure that:
 +
+ifdef::restricted[]
 ifdef::openshift-rosa,openshift-enterprise[]
 * If you are using {FeatureName} with AWS Secure Token Service (STS), in the *role ARN* field, enter the ARN role copied from the last step of the _Obtaining a role Amazon Resource Name for Security Token Service_ procedure.
 endif::[]
```

Comment:
I think it'll be fine without it and it's up to you, but since you have multiple ifdefs right here, it might be good to clarify which attributes close this one out:
```suggestion
endif::openshift-rosa,openshift-enterprise[]
```